src/Catalyst.Abstractions/Kvm/Models/BlockParameters.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using Nethermind.Core.Json;
+
+namespace Catalyst.Abstractions.Kvm.Models
+{
+    public class BlockParameter
+    {
+        public static BlockParameter Earliest = new BlockParameter(BlockParameterType.Earliest);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4411902)